Bio/Affy/CelFile.py

```diff
@@ -185,7 +318,3 @@ def read(handle):
         else:
             continue
     return record
-
-if __name__ == "__main__":
-    from Bio._utils import run_doctest
-    run_doctest()
```

Comment:
I have migrated some of these test cases to test_Affy.py. I didn't realise these were getting run by CI, and I'll put this back in.